src/main/java/github/weichware10/toolbox/Main.java

```diff
@@ -1,71 +1,149 @@
-import javafx.event.EventHandler;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
 import javafx.scene.Scene;
 import javafx.scene.control.TextArea;
 import javafx.scene.image.Image;
+import javafx.scene.text.Text;
+import javafx.stage.FileChooser;
 import javafx.stage.Stage;
-import javafx.stage.WindowEvent;
+import javafx.stage.FileChooser.ExtensionFilter;
 
 /**
  * Toolbox GUI.
  *
  */
 public class Main extends Application {
+    private Stage primaryStage;
+    static private ConfigClient configClient;
+    static private DataBaseClient dataBaseClient;
+
     public static void main(String[] args) {
         launch(args);
     }
 
     @Override
     public void start(Stage primaryStage) throws Exception {
 
-        // Konsole zum Debuggen
-        boolean debug = false;
-        if (debug) {
-            Stage logStage = new Stage();
-            TextArea textArea = new TextArea();
-            Console console = new Console(textArea);
-            PrintStream ps = new PrintStream(console, true);
-            System.setOut(ps);
-            System.setErr(ps);
-            Scene terminal = new Scene(textArea);
-            logStage.setScene(terminal);
-            logStage.show();
-        }
+        resetDataBaseConnection();
+
+        this.primaryStage = primaryStage;
+
+        primaryStage.setMinWidth(300);
+        primaryStage.setMinHeight(300);
+
+        FXMLLoader loader = new FXMLLoader(getClass().getResource("App.fxml"));
 
+        Parent root = loader.load();
 
-        //startet die erste Szene
-        Startbildschirm.display(primaryStage);
+        AppController controller = loader.getController();
+        controller.setMain(this);
 
-        // Event welches beim schließen eines Fensters aufgerufen wird
-        primaryStage.setOnCloseRequest(new EventHandler<WindowEvent>() {
-            @Override
-            public void handle(WindowEvent event) {
-                // Wir kümmern uns selber um das schließen
-                event.consume();
-                // Fenster schließen, ja oder nein?
-                closeProgramm(primaryStage);
-            }
-        });
+        Scene scene = new Scene(root, 300, 275);
 
-        //Zeigt das Fenster
         primaryStage.getIcons().add(new Image("app-icon.png"));
+        primaryStage.setTitle("Toolbox");
+        primaryStage.setScene(scene);
         primaryStage.show();
     }
 
-    //Nachfrage ob Programm wirklich beendet werden soll
-    private void closeProgramm(Stage window) {
+    // Konsole zum Debuggen
+    /*
+     * boolean debug = false;
+     * if (debug) {
+     * Stage logStage = new Stage();
+     * TextArea textArea = new TextArea();
+     * Console console = new Console(textArea);
+     * PrintStream ps = new PrintStream(console, true);
+     * System.setOut(ps);
+     * System.setErr(ps);
+     * Scene terminal = new Scene(textArea);
+     * logStage.setScene(terminal);
+     * logStage.show();
+     * }
+     */
+
+     /**
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.CommentsIndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Block comment has incorrect indentation level 5, expected is 4, indentation should be the same level as line 78.